Week3/homework/step2-3.js

```diff
@@ -28,11 +27,11 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 
 // Use a 'do...while' loop
 function repeatStringNumTimesWithDoWhile(str, num) {
-  // eslint-disable-next-line
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  do {
```

Comment:
What if `num <= 0`? Then the `do` loop body will run once anyway because the loop condition will only be executed at the end of the loop. This makes the `do-while` loop unsafe and is best replaced by a regular `while` loop which tests the condition before entering the loop body. We asked to use the `do-while` loop here to demonstrate this problem. What code can you add to prevent the `do-while` loop from running if `num <= 0`?